test/integration/wallet.test.js

```diff
@@ -0,0 +1,133 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const authService = require('../../services/authService')
+
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+const usersUtils = require('../../utils/users')
+
+const userData = require('../fixtures/user/user')()
+const { walletBodyKeys, walletKeys, walletDataKeys } = require('../fixtures/wallet/wallet')
+
+const defaultUser = userData[0]
+const newUser = userData[3]
+const superUser = userData[4]
+
+const config = require('config')
+const cookieName = config.get('userToken.cookieName')
+
+chai.use(chaiHttp)
+
+describe('Wallet', function () {
+  let authToken
+  let userId
+  let userName
+
+  beforeEach(async function () {
+    userId = await addUser(defaultUser)
+    authToken = authService.generateAuthToken({ userId })
+    userName = await usersUtils.getUsername(userId)
+  })
+
+  afterEach(async function () {
+    await cleanDb()
+  })
+
+  describe('Check /wallet', function () {
+    it('Should return wallet information', function (done) {
+      chai
+        .request(app)
+        .get('/wallet')
+        .set('cookie', `${cookieName}=${authToken}`)
+        .end((error, response) => {
+          if (error) {
+            return done(error)
+          }
+
+          expect(response).to.have.status(200)
+          expect(response.body).to.be.a('object')
+          expect(response.body).to.have.all.keys(...walletBodyKeys)
+          expect(response.body.message).to.be.equal('Wallet returned successfully for user')
+          expect(response.body.wallet).to.be.a('object')
+          expect(response.body.wallet).to.have.all.keys(...walletKeys)
+          expect(response.body.wallet.data).to.have.all.keys(...walletDataKeys)
+
+          return done()
+        })
+    })
+
+    it('Should return the user their own wallet with 1000 dineros loaded', function (done) {
+      chai
```

Comment:
Let's add one more testcase when the cookie is missing, 401 expected

Reply:
Is this needed now? or will the below one do?

Reply:
@swarajpure let me know

Reply:
![image](https://user-images.githubusercontent.com/10326494/123144522-5e1ca900-d479-11eb-99e2-a519272f6475.png)

Updated the same